xarray/plot/plot.py

```diff
@@ -175,9 +175,13 @@ def line(darray, *args, **kwargs):
         Axis on which to plot this figure. By default, use the current axis.
         Mutually exclusive with ``size`` and ``figsize``.
     hue : string, optional
-        Coordinate for which you want multiple lines plotted (2D inputs only).
+        Coordinate for which you want multiple lines plotted
+        (2D DataArrays only).
     x : string, optional
-        Coordinate for x axis.
+        1D and 2D DataArrays: Coordinate for x axis.
+    y : string, optional
+        1D DataArray: Can be coordinate name or DataArray.name
+        2D DataArray: Coordinate for y axis.
```

Comment:
I'm not sure this is really helping. I'd prefer to stick to the simple ``Coordinate for x/y axis.``